Gemfile

```diff
@@ -84,3 +84,5 @@ gem "kaminari", "~> 1.2"
 
 gem "sanitize", "~> 6.0"
 gem "aws-sdk-s3", require: false
+
+gem "bcrypt", "~> 3.1"
```

Comment:
in hindsight the adding of a gem should be on it's own branch

Reply:
Could have been done first. I'd say the more relevant take away is that `bcyrpt` was already in your Gemfile.

```
# Use Active Model has_secure_password [https://guides.rubyonrails.org/active_model_basics.html#securepassword]
# gem "bcrypt", "~> 3.1.7"
```